go/types/list.go

```diff
@@ -99,6 +117,7 @@ func (l List) Get(idx uint64) Value {
 type MapFunc func(v Value, index uint64) interface{}
 
 func (l List) Map(mf MapFunc) []interface{} {
+	// TODO: This is bad API. It should have returned another List.
```

Comment:
How about using `Deprecated`? For example: https://golang.org/src/net/http/httputil/persist.go?s=918:1235#L31